src/main/java/javaintern/constant/ReservationState.java

```diff
@@ -0,0 +1,5 @@
+package javaintern.constant;
+
+public enum ReservationState {
+	INITIAL, CONFIRMED, FULLY_PAID, CANCELLED
```

Comment:
change style according:
State of record (Initial, Confirmed, FullyPaid, Cancelled)